docs/modules/ROOT/pages/how-to/manage-projects-and-namespaces.adoc

```diff
@@ -0,0 +1,38 @@
+= Manage Projects and Namespaces
+
+This page explains the various mechanisms available to create OpenShift Projects in APPUiO Cloud.
+
+== Creating Projects
+
+Through the web console:: Click on the btn:[Create Project] button and fill the form to create a new project.
+
+Through the Command Line:: Use the `oc new-project` command.
+
+Through the API:: Use the `ProjectRequest` resource to create a new project.
+
+== Project Ownership
+
+OpenShift Projects will always be assigned to the default organization specified by the `appuio.io/organization` label. Check the xref:explanation/organizations-and-rbac.adoc[Organizations and RBAC] page for more information.
+
```

Comment:
Inform the reader by using admonition about the issue with the undefined default organization. Hint that the workaround is to use the new namespace with explicit organization label approach.

Reply:
Done